### DISCUSSION TITLE
What is the most cheapest on RAM index?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [milvus-io/milvus#20688] [Q&A and General discussion] [answered]
What is the most cheapest on RAM index?

**Comment by yhmo** [comment, marked as answer]
IVF_PQ or IVF_SQ8, 20%~30% memory usage compare to IVF_FLAT

**Reply by martinenkoEduard** [reply]
does IVF_FLAT consume more memory than just Flat?

**Reply by xiaofan-luan** [reply]
> does IVF_FLAT consume more memory than just Flat?

Should almost be the same. Maybe slightly more
